core/modules/date/date.info

```diff
@@ -1,6 +1,6 @@
 name = Date
 description = Makes date/time fields available.
-package = Date/Time
+package = Fields
```

Comment:
Oops, not relevant to this PR. I think we already made this change in 1.x? Maybe not.